docs/event-mapping/xAPI_mapping.rst

```diff
@@ -885,3 +885,24 @@ id                                                                          <LMS
 objectType                                                                  Activity
 definition [ type ]                                                         http://id.tincanapi.com/activitytype/discussion
 =========================================================================== =======================================================
+
+edx.certificate.created
+=============================
+
+=========================================================================== =======================================================
+xAPI Key                                                                    Value
+=========================================================================== =======================================================
+``Actor``
+objectType                                                                  Agent
+account [ homePage ]                                                        <LMS_ROOT_URL>
+account [ name ]                                                            <external_id[ XAPI ]>
+``Verb``
+id                                                                          http://id.tincanapi.com/verb/earned
+display [ en-US ]                                                           earned
+``Object``
+id                                                                          <LMS_ROOT_URL>/certificates/<data [ certificate_id ]>
+objectType                                                                  Activity
+definition [ type ]                                                         https://www.opigno.org/en/tincan_registry/activity_type/certificate
```

Comment:
This [URL](https://www.opigno.org/tincan_registry/activity_type/certificate
) shows me 404

Reply:
me too, hahaha This is part of the requirements of the NELC guide  and this is also mentioned in the [tincanapi page](https://registry.tincanapi.com/#home/activityTypes) 

https://registry.tincanapi.com/#uri/activityType/413